src/port1.0/portconfigure.tcl

```diff
@@ -527,9 +527,17 @@ proc portconfigure::get_compiler_fallback {} {
                 return {gcc-4.0}
             }
         } elseif {[vercmp $xcodeversion 3.0] >= 0} {
-            return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-clang-3.4 macports-clang-3.3}
+            if {${configure.build_arch} eq "ppc" || ${configure.build_arch} eq "ppc64"} {
+                return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-gcc-6  macports-gcc-7}
+            } else {
+                return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-clang-3.4 macports-clang-3.3}
+            }
         } else {
-            return {apple-gcc-4.2 gcc-4.0 gcc-3.3 macports-clang-3.3}
+            if {${configure.build_arch} eq "ppc" || ${configure.build_arch} eq "ppc64"} {
+                return {apple-gcc-4.2 gcc-4.0 macports-gcc-6 macports-gcc-7}
```

Comment:
Why did you remove `gcc-3.3` from the list for 10.4 PowerPC?

Reply:
just thought nobody cared about gcc-3.3 for anything anymore, and we would never want it to be called in. I can add it back if you know of something that builds with it.

Reply:
I care about the inconsistency between Intel and PowerPC. I haven't seen a ticket or mailing list thread discussing dropping gcc-3.3, but if we want to drop it, shouldn't we always drop it, and not just on PowerPC?

Reply:
True. will fix.